Proof-of-Stake.md

```diff
@@ -118,10 +118,10 @@ Aside from costing a massive amount of money, the consensus mechanism does have
 
 ### The Downsides
 
-Not all is rainbows and unicorns in this world however, and with it's many advantages, PoS does have a few cons that continue to make consensus mechanisms an active field of research and innovation.
+With its many advantages, PoS does have a few cons that continue to make consensus mechanisms an active field of research and innovation.
```

Comment:
I would appreciate it if some portion of this could be left without straight up jumping to the point. 

Maybe you can say something like : 
`Not all are Rainbows and Unicorns, unfortunately. PoS does have....`

Reply:
This can also be left as it is @SlavaTimbaliuc /

Reply:
The `in this world`, is a bit too much lol. Still don't wanna completely cut if off. At least that's what I think. @ljcutts

Reply:
The reason why I think it could be left as is, is because it adds to the humor of the statement.

Reply:
> The reason why I think it could be left as is, is because it adds to the humor of the statement.

Exactly why I said a portion should be left. The `in the world` portion sounds a bit dark, which is why I asked him to omit that and keep the rest, but if you feel like we should keep all, sure.

Reply:
Left the rainbows comment, removed "in this world"